CMakeLists.txt

```diff
@@ -2,10 +2,14 @@ cmake_minimum_required(VERSION 3.25)
 project(onnxruntime_extensions LANGUAGES C CXX)
 
 # set(CMAKE_VERBOSE_MAKEFILE ON)
-if(NOT CMAKE_BUILD_TYPE)
+if(NOT WIN32 AND NOT CMAKE_BUILD_TYPE)
   message(STATUS "Build type not set - using RelWithDebInfo")
   set(CMAKE_BUILD_TYPE "RelWithDebInfo" CACHE STRING "Choose build type: Debug Release RelWithDebInfo." FORCE)
 endif()
+if (OCOS_ENABLE_STATIC_LIB) 
+    add_compile_options(/MT$<$<CONFIG:Debug>:d>)
```

Comment:
what's about other build-flavor, like release, relwithdebinfo?

Reply:
for all other build types it'll link with /MT only. I'll build with RelWithDebInfo and fix as needed.